apps/entry/filter_set.py

```diff
@@ -143,6 +148,13 @@ class Meta:
             },
         }
 
+    # def lead_title_filter(self, queryset, name, value):
+    #     if value.strip():
+    #         return queryset.annotate(
+    #             idx=StrIndex(Lower('lead__title'), Value(value.lower()))
+    #         ).filter(idx__gt=0).order_by('idx', 'lead__title')
+    #     return queryset
+
```

Comment:
Let's not keep commented code. If you think it's good to keep this, let's create a 2 commit w/o this.

Reply:
I second @thenav56